README.md

```diff
@@ -71,6 +84,12 @@ module "ipa_master" {
 | master | The IPA master EC2 instance. |
 | server_security_group | The IPA server security group. |
 
+## Notes ##
+
+Running `pre-commit` requires running `terraform init` in every directory that
+contains Terraform code. In this repository, these are the main directory and
+every directory under `examples/`.
+
```

Comment:
I believe you would need to run `terraform init` in the `dns` directory as well.

Reply:
See commit 4a71226.

Reply:
Thanks as always, but a typo emerged in that commit: "a well as" -> "as well as" (sorry).

Reply:
See commit 5c8350f.

Reply:
Appreciate you 👍